yb-voyager/cmd/exportData.go

```diff
@@ -90,35 +92,33 @@ func exportDataOffline() bool {
 	// defer cancel()
 
 	var tableList []string
+	var finalTableList []string
```

Comment:
Instead of introducing a new variable `finalTableList` can we just `tableList` var, and just update/filter the complete `tableList` based on whether the `--table-list` or `--exclude-table-list` is provided.

Reply:
If you just update the `tableList` var by excluding the ones, will there still be a need to update any code at other places?